llvm/test/CodeGen/EraVM/cgp-uadd-overflow-wrong-match.ll

```diff
@@ -0,0 +1,50 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 2
```

Comment:
Perhaps it's better to stop after CGP and check for the intrinsic instead of running all codegen pipeline.

Reply:
Makes sense. Added test that runs only CGP and matched IR.